src/test-smoke/groovy/DummyTest.groovy

```diff
@@ -0,0 +1,9 @@
+import spock.lang.Specification
+
+class DummyTest extends Specification {
+
+    def "dummy test"() {
+        expect:
+        true
```

Comment:
You could indent `true` to be compilant with other test specifications  - see https://solidsoft.wordpress.com/2014/05/20/more-compact-automatic-spock-specification-formatting-in-idea-13/ how to do it automatically in Idea.